### ISSUE TITLE
Specify whether false-positives are expected in docs

### ISSUE THREAD

**Issue by rcoh** [rust-lang-nursery/rust-clippy#2680]
I've run into lints that are false positives on clippy. I think it would be helpful if the lints in the `clippy` collection showed something like:

```We expected this lint, `lint_name_here` to have no false positives. If you found one, please report it! https://...../issues/false-positive-template
```

This could be both in the online docs (perhaps something like `No false positives. Found One?` in the header line of the lint, as well as in the command line output.

Thoughts?

**Comment by oli-obk** [comment]
Currently we assume that any lints which don't state that they have false positives, don't have false positives. But we could state that in the docs, indeed.

Essentially everywhere where the docs say `None`, we could paste the text you mentioned along with the hyperlink. Bonus points for doing that via the python script, so we can still just write `None`
